deploy/crds/toolchain_v1alpha1_nstemplateset.yaml

```diff
@@ -6,6 +6,16 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: nstemplatesets.toolchain.dev.openshift.com
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.tierName
```

Comment:
should this file change be part of a separate PR to keep the `master` branch in sync with the `api` module?

Reply:
Initially, started with similar idea to merge both api and member-operator together but later found member-operator need to point to latest dependency of `api` so api_pr need to merge early.

Reply:
agree with @xcoulon, it would be better to make a separate PR.

Reply:
Next time will make sure, this time anyway this is about to over and will merge soon anyway :-)